package.json

```diff
@@ -55,13 +56,12 @@
     "jest-cli": "^24.9.0",
     "prettier": "^1.18.2",
     "pretty-quick": "^2.0.0",
+    "ramda": "^0.26.1",
```

Comment:
For the purposes of packmap within OpenMRS:

`dependencies` = in-browser dependencies
`devDependencies` = everything else